src/sas/qtgui/Perspectives/Fitting/UI/SmearingWidgetUI.ui

```diff
@@ -101,6 +101,9 @@
           <property name="title">
            <string>Accuracy</string>
           </property>
+          <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Higher accuracy is very expensive. Use it with care!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+          </property>

```

Comment:
Maybe we can have a custom label (there's plenty of room on this control!) which will display the above message explicitly rather than in a tooltip, whenever users choose higher accuracy?
This is enhancement rather than issue with the current code, so feel free to make it into a separate GH issue.

Reply:
Why not? I think that a user will see the tooltip whenever changing the accuracy, but I don't know if he/she will read it or just ignore it :-) So I'll let Paul B decide what's best.

Reply:
Sorry am looking at your code changes vs what I had to do in 4.x (which was apparently only a year and a bit ago -- seems like forever.

Not sure I should be the decider.  Was going to do some functionality testing after I compare the codes and see but my first guess would be that @rozyczko suggestion of making it a more permanent warning when high is chosen would be good.  I'd still leave the tool tip on that case though since there are 4 levels as I recall that get progressively more costly and @rozyczko suggestion would only be for the worst case?

What does Lionel think? he is testing right? 😄  speaking of which maybe we can make an installable from this PR.. I cannot remember the 5.x magic runes need to look them up.